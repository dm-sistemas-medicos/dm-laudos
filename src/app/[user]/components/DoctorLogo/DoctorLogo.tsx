import Image from "next/image";
import Link from "next/link";

const DoctorLogo = ({ user }: { user: string }) => (
  <Link href={`/${user}`}>
    <Image src={"/logo.png"} alt="" width={400} height={123} priority={true} />
  </Link>
);

export default DoctorLogo;
