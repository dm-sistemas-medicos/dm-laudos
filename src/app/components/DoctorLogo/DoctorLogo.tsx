"use client";

import Image from "next/image";
import Link from "next/link";

const DoctorLogo = () => (
  <Link href="/">
    <Image src={"/logo.png"} alt="" width={400} height={123} priority={true} />
  </Link>
);

export default DoctorLogo;
