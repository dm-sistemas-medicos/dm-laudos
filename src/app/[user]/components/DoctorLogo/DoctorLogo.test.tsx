import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import DoctorLogo from "./DoctorLogo";

describe("DoctorLogo component", () => {
  it("renders the component", () => {
    const { container } = render(<DoctorLogo />);

    expect(container).toMatchSnapshot();
  });
});
