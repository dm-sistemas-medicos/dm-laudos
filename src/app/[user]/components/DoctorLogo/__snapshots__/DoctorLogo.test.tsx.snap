// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoctorLogo component renders the component 1`] = `
<div>
  <a
    href="/"
  >
    <img
      alt=""
      data-nimg="1"
      decoding="async"
      height="123"
      src="/_next/image?url=%2Flogo.png&w=828&q=75"
      srcset="/_next/image?url=%2Flogo.png&w=640&q=75 1x, /_next/image?url=%2Flogo.png&w=828&q=75 2x"
      style="color: transparent;"
      width="400"
    />
  </a>
</div>
`;
